sql/core/src/test/scala/org/apache/spark/sql/execution/BroadcastExchangeSuite.scala

```diff
@@ -82,7 +83,7 @@ class BroadcastExchangeSuite extends SparkPlanTest
       val events = jobEvents.toArray
       val hasStart = events(0).isInstanceOf[SparkListenerJobStart]
       val hasCancelled = events(1).asInstanceOf[SparkListenerJobEnd].jobResult
-        .asInstanceOf[JobFailed].exception.getMessage.contains("cancelled job tag")
+        .asInstanceOf[JobFailed].exception.getMessage.contains("Could not execute broadcast")
```

Comment:
This is changed because now on timeout, `BroadcastExchangeExec` will now print out the exception, not a generic statement such as "cancelled job tag". See [this line](https://github.com/apache/spark/blob/0444ba32156e424ee6d902c3f4a4c44a07993273/sql/core/src/main/scala/org/apache/spark/sql/execution/exchange/BroadcastExchangeExec.scala#L241)